Services/CO.CDP.Organisation.WebApi/appsettings.json

```diff
@@ -8,6 +8,7 @@
       }
     }
   },
+  "OrganisationApiHostUrl": "http://localhost:8082",
```

Comment:
"HostUrl" doesn't make much sense - it's either one or the other. Can we name the option `OrganisationApiUrl` or `OrganisationWebApiUrl` to be consistent with `OrganisationAppUrl` we already have?

Reply:
done